schema/AdamSensors.yaml

```diff
@@ -8,7 +8,7 @@ properties:
   adam_ip:
     description: IP of the ADAM controller.
     type: string
-    default: "140.252.33.160"
+    default: "140.252.32.110"
```

Comment:
This is probably not a good default for a configuration.... I think you want to have something like 127.0.0.1 (localhost), which you will use for the simulators.